docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md

```diff
@@ -19,6 +19,8 @@ seo:
 Follow these step-by-step instructions to deploy ENBUILD locally for testing.
 
 ## Prerequisites
+Before you begin, ensure that you have the following prerequisites in place:
```

Comment:
why do we need docker ?

Reply:
Docker is the containerization platform that manages the k3d clusters. When I tried running the clusters without docker they wouldn't start.

Reply:
yeah, so we have covered Kubernetes as Prerequisites,  how user creates Kubernetes is upto them. 
k3d is one option.